WooCommerce/Classes/ViewRelated/BottomSheet/ListSelector/BottomSheetListSelectorViewProperties.swift

```diff
@@ -0,0 +1,5 @@
+/// Properties for the bottom sheet list selector view.
+///
+struct BottomSheetListSelectorViewProperties {
+    let title: String?
+}
```

Comment:
yep! updated here: https://github.com/woocommerce/woocommerce-ios/pull/2155/commits/72e8b30acc388f13e98e774bbc77dfca45084dca

Reply:
after updating the use case PR, I encountered this error since `BottomSheetListSelectorViewController` has generics:

```
Expression type 'BottomSheetListSelectorViewController<_, _, _>.ViewProperties' is ambiguous without more context
```

I reverted the changes in the comment above in https://github.com/woocommerce/woocommerce-ios/pull/2155/commits/56193581e35403044142ffbe84d1c1858662c86c - lemme know if you have any ideas for making a nested struct under a generic class!